Dockerfile

```diff
@@ -0,0 +1,41 @@
+# STAGE base: Development environment
+FROM node:10-alpine@sha256:77c898d0da5e7bfb6e05c9a64de136ba4e03889a72f3c298e95df822a38f450d \
+	AS base
+
+WORKDIR /devswag
+
+ENV GULP_LISTEN_HOST=0.0.0.0
+ENV GULP_LISTEN_PORT=8080
+
+COPY package.json package-lock.json ./
+
+RUN set -x \
+	# Install npm dependencies
+	&& npm ci \
```

Comment:
ci shouldn't be used for local development, right?

Reply:
Since we want to install the dependencies from the lockfile without editing it I think it's the best way of installing deps.